showcases/react-showcase/package.json

```
@@ -13,7 +13,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@db-ui/foundations": "*",
+    "@db-ui/foundations": "file:../../packagesi/foundations",
```

@nmerget Is there a typo in the path ("packagesi"), and in the other files as well ?
```suggestion
    "@db-ui/foundations": "file:../../packages/foundations",
```

---

fixed